proof/infoflow/Ipc_IF.thy

```diff
@@ -601,6 +601,7 @@ lemma send_signal_reads_respects:
                         set_thread_state_runnable_equiv_but_for_labels get_simple_ko_wp
                         gts_wp update_waiting_ntfn_equiv_but_for_labels
                         blocked_cancel_ipc_nosts_equiv_but_for_labels
+            | wp_pre0
```

Comment:
This could probably be removed if we manage to add `wp_pre` to `wp (once)`. I should maybe leave a FIXME saying that.

Reply:
yes, please